packages/react-components/src/components/VideoGallery/Layout.ts

```diff
@@ -28,7 +28,7 @@ export interface LayoutProps {
   screenShareComponent?: JSX.Element;
   /**
    * Maximum number of participant remote video streams that is rendered.
-   * @defaultValue 4
+   * @defaultValue 9
```

Comment:
Can we maintain the same defaultValue for # of streams in VideoGallery and the Composite? Can we bring up this question in our Feature meeting if we don't know which value to snap to and why?